src/v/rpc/netbuf.cc

```diff
@@ -53,7 +53,7 @@ ss::scattered_message<char> netbuf::as_scattered() && {
     _out.prepend(header_as_iobuf(_hdr));
 
     // prepare for output
-    return iobuf_as_scattered(std::move(_out));
+    co_return iobuf_as_scattered(std::move(_out));
```

Comment:
Please update the commit message: when we say "coroutinize" we usually mean that an existing future-returning function is being refactored to use coroutines instead of `.then()` style: this commit is more about turning previously synchronous code into async code

Reply:
Updated commit message to better reflect the purpose of the commit.